packages/search-engine/src/direflow-components/search-engine/ResultView.tsx

```diff
@@ -12,7 +12,7 @@ const ResultView= () => {
               dataField='tweet'
               className='result-list'
               react={{
-                and: [ 'party-filter', 'mainSearch']
+                and: [ 'party-filter', 'mainSearch', 'date-filter']
```

Comment:
the cases are a bit inconsistent, your could either use `snake-case` or `camelCase`